pkg/cli/debug_merge_logs.go

```diff
@@ -541,7 +555,9 @@ func (s *fileLogStream) error() error        { return s.err }
 
 // seekToFirstAfterFrom uses binary search to seek to an offset after all
 // entries which occur before from.
-func seekToFirstAfterFrom(f *os.File, from time.Time, editMode log.EditSensitiveData) (err error) {
+func seekToFirstAfterFrom(
```

Comment:
Now that we're buffering a whole file in RAM (something I was actively trying to avoid when I wrote this), we can have this just return `([]byte, error)`.